setup.py

```diff
@@ -1,8 +1,18 @@
 """Setup script for lasio"""
 
+import lasio
 from setuptools import setup
 import os
 
+__init__file = './lasio/__init__.py'
```

Comment:
Better to use `os.path.join` here. And I think you don't need the `.` part on the start, since open operates from the current working directory by default anyway